### ISSUE TITLE
Discord 'Quests' (ads)

### ISSUE THREAD

**Issue by nshjck** [CroissantDuNord/discord-adblock#27]
Is there plans for an update hiding quests from the app?

**Comment by sumsare** [comment]
> Is there plans for an update hiding quests from the app?

![image](https://github.com/CroissantDuNord/discord-adblock/assets/69218230/dd63ff08-0373-4f1e-9f6a-bca4151870de)
there's this setting under 'Privacy & Safety' settings for now. Who knows when they'll decide to disable it and force the quests on everyone though...

**Comment by nshjck** [comment]
that setting only changes what they reccomend you, it doesn't actual stop you from seeing them

**Comment by sumsare** [comment]
> that setting only changes what they reccomend you, it doesn't actual stop you from seeing them

Oh, it managed to hide it for me (I had one about genshin impact, which I don't play anymore but had in my registered games list, I did delete every game I had saved there as well manually) but maybe tomorrow it'll be back with a different game. It's a shame it didn't work for you :( though this "quests" feature was what made me look up an adblock for discord in the first place. I do agree it would be nice to have this adblock disable the quests as well :)

**Comment by nshjck** [comment]
yeah I was thinking having the adblock disable them would be nice, maybe they didn't disappear because I didn't restart the app

**Comment by nixRidge** [comment]
I'll definitely block this if I see it. Sounds annoying as hell.

**Comment by rom4ster** [comment]
What good is this thing if it does not block that? Literally only good if you dont have nitro. Block that shit please..

**Comment by nixRidge** [comment]
I can only block it if I see it myself. Quests have not appeared in my client yet, @rom4ster 

Someone can contribute a rule if they wish.

**Comment by nshjck** [comment]
afaik they only show up on the right side of the friends list page if your friend is playing a game that has a quest associated

**Comment by rom4ster** [comment]
> afaik they only show up on the right side of the friends list page if your friend is playing a game that has a quest associated

Mine show up on the bottom left
